x-pack/plugins/alerts/README.md

```diff
@@ -498,42 +409,6 @@ You can use the `registerNavigation` api to specify as many AlertType specific h
 
```

Comment:
I think we should add a link in here to the asciidocs making it clear that there's a whole load of APIs that *aren't* listed here.
If I'm a new developer who isn't familiar with the public docs, I might not know I have a whole REST api suite available to me.